raven/build.gradle

```diff
@@ -0,0 +1,55 @@
+plugins {
+    id 'war'
+    id 'maven'
+    id 'maven-publish'
+    id 'com.jfrog.bintray' version '1.8.4'
+    id 'checkstyle'
+}
+
+repositories {
+    jcenter()
+    mavenLocal()
+}
+
+apply from: '../opencadc.gradle'
+
+sourceCompatibility = 1.8
+
+group = 'org.opencadc'
+
+war {
+    from(System.getenv('RPS') + '/resources/') {
+        include 'swagger-ui/'
+    }
+    // expect a private key in this location
+    from(System.getenv('A') + '/etc/') {
+        include 'DelegationTokenKey.priv'
+        rename 'DelegationTokenKey.priv', 'InventoryPriv.key'
+        into 'WEB-INF/classes'
+    }
+}
+
+dependencies {
+    compile 'log4j:log4j:[1.2,)'
+    compile 'org.opencadc:cadc-util:[1.2,)'
+    compile 'org.opencadc:cadc-log:[1.0,)'
+    compile 'org.opencadc:cadc-registry:[1.2.1,)'
+    compile 'org.opencadc:cadc-vosi:[1.0,)'
+    compile 'org.opencadc:cadc-rest:[1.0,)'
+    compile 'org.opencadc:cadc-cdp:[1.0,)'
+    compile 'org.opencadc:cadc-gms:[1.0.0,)'
+    compile 'org.opencadc:cadc-inventory:[0.1,)'
+    compile 'org.opencadc:cadc-inventory-db:[0.1,)'
+    compile 'org.opencadc:cadc-storage-permissions:[0.1,)'
+    compile 'org.opencadc:cadc-vos:[1.1.8,)'
```

Comment:
cadc-access-control* dependencies should be runtime so we don't write code against that API